docs: Doc Updates for 1.11 (#2487)
kubeshop/monokle: docs/git-integration.md

# Git Integration for DevOps & GitOps

> Git is integrated into Monokle to tackle GitOps seamlessly within the application. If you work within a DevOps culture our Git integration provides advantages for you and your team by keeping a history of changes made to your applications stored in Git to ensure access to a reliable audit trail of your changes.

Reviewer: Monokle integrates with Git to provide support for Git based flows, including GitOps. If you using DevOps techniques and culter, our Git integratrion provides a history of changes made to your applications, with a full audit trail of your changes.

> Git is integrated into Monokle to tackle GitOps seamlessly within the application. If you work within a DevOps culture our Git integration provides advantages for you and your team by keeping a history of changes made to your applications stored in Git to ensure access to a reliable audit trail of your changes.

Author: ```suggestion
Monokle integrates with Git to provide support for Git based flows, including GitOps. If you using DevOps techniques and culture, our Git integration provides a history of changes made to your applications, with a full audit trail of your changes.
```

## **Getting Started with a New Project: Clone a Git repository**

> You can use a Git repository URL to start a new project. The repository will be cloned to the selected location and then opened into Monokle.

Reviewer: You can start a new clone form a Git repository URL. The repo will be cloned locally and then opened into Monokle

Author: Changed.

> You can use a Git repository URL to start a new project. The repository will be cloned to the selected location and then opened into Monokle.

Author: ```suggestion
You can start a new clone from a Git repository URL. The repo will be cloned locally and then opened into Monokle.
```

![Clone Git Repo](img/clone-a-git-repo-on-startup-1.11.png)

> This allows multiple colleagues to debug, validate, and diff all of your configuration files taking full advantage of all of the features of Monokle to achieve your desired state while also maintaining clear control of your version changes.

Reviewer: You can use the collaboration support of Git to debug, validate, and diff all your configuration files to define and store your desired state with a full audit trail. All features of Monokle are already or in the process to fully function with Git.

Author: Changed.

> This allows multiple colleagues to debug, validate, and diff all of your configuration files taking full advantage of all of the features of Monokle to achieve your desired state while also maintaining clear control of your version changes.

Author: ```suggestion
You can use the collaboration support of Git to debug, validate, and diff all your configuration files to define and store your desired state with a full audit trail. All features of Monokle are already or in the process to fully function with Git.
```
